build_tools/cmake/riscv.toolchain.cmake

```diff
@@ -23,14 +23,15 @@ set(RISCV_HOST_TAG linux)
 set(RISCV_TOOL_PATH "$ENV{HOME}/riscv" CACHE PATH "RISC-V tool path")
 
 set(RISCV_TOOLCHAIN_ROOT "${RISCV_TOOL_PATH}/toolchain/clang/${RISCV_HOST_TAG}/RISCV" CACHE PATH "RISC-V compiler path")
+set(RISCV_TOOLCHAIN_PREFIX "" CACHE STRING "RISC-V compiler prefix")
```

Comment:
Please set a default, and also update https://github.com/openxla/iree/blob/main/build_tools/cmake/build_riscv.sh with the proper prefix

Reply:
Ok, I'll set a default value and update build_riscv.sh